src/bindings/event.ts

```diff
@@ -48,20 +48,23 @@ export const bindEvents = async (context: Context): Promise<void> => {
   logger.info("Creating adapters for supabase, telegram, twitter, etc...");
   adapters = createAdapters(botConfig);
 
-  // Validate payload
-  const validate = ajv.compile(PayloadSchema);
-  const valid = validate(payload);
-  if (!valid) {
-    logger.info("Payload schema validation failed!!!", payload);
-    logger.warn(validate.errors!);
-    return;
-  }
+  // Skip validation for installation event
+  if (eventName !== GithubEvent.INSTALLATION_ADDED_EVENT) {
```

Comment:
Can you explain to me what is happening here?

Reply:
I think we need the wildcard processor but I'm not 100% sure @0xcodercrane

Reply:
> I think we need the wildcard processor but I'm not 100% sure @0xcodercrane

`checkBountiesToUnassign`, `collectAnalytics`, `checkWeeklyUpdate` is not something we want to run on install.. 

Its just been setup, no  bounties to unassign, no analytics and no weekly update.. so it'd often break there

Reply:
Here @0xcodercrane